vizro-core/tests/unit/vizro/models/_controls/test_parameter.py

```diff
@@ -78,6 +80,26 @@ def test_target_and_title_generation_valid(self, test_input, title):
         assert parameter.targets == ["scatter_chart.x"]
         assert parameter.selector.title == title
 
+    @pytest.mark.parametrize(
+        "test_input",
+        [
+            (vm.Slider(min=0, max=1, value=0.8)),
+            (vm.RangeSlider(min=0, max=1, value=[0.2, 0.8])),
+            (vm.Checklist(options=["lifeExp", "gdpPercap", "pop"], value=["lifeExp"])),
+            (vm.Dropdown(options=["lifeExp", "gdpPercap", "pop"], multi=False, value="lifeExp")),
+            (vm.RadioItems(options=["lifeExp", "gdpPercap", "pop"], value="lifeExp")),
+        ],
+    )
+    def test_actions_generation_valid(self, test_input):
+        parameter = Parameter(targets=["scatter_chart.x"], selector=test_input)
```

Comment:
Does this actually work?

You are providing numeric and categorical parameters to the same target. I am surprised that this doesn't throw an error?